content/docs-v1/multiformat-output.md

```diff
@@ -157,9 +157,123 @@ For example, this would hide all `video` elements in the print output:
 
 Some of the CSS used in styling the PDF is from the CSS Paged Media Specification. This is a set of CSS rules designed specifically to style things in a page-like manner, including controlling left and right page rules, page numbering, and running feet and heads. There is good information about this in [Paged.js’s documentation](https://pagedjs.org/documentation/) as well as [in PrinceXML’s documentation](https://www.princexml.com/doc/paged/). For the most part, the CSS rules documented for one are also applicable to the other. The one primary exception are any CSS attributes that begin with a custom `-prince` name.
 
+Custom CSS can be particularly useful in addressing common layout issues in the PDF output like extra white space or awkward breaks at the end of columns and pages.
+
+The following examples illustrate common scenarios and offer sample CSS that can be copy and pasted into your project's `content/_assets/styles/custom.css` file. Many require using PrinceXML to generate your PDF rather than Paged.js. Note too that these styles have been wrapped in a print `@media` query to ensure they only affect the PDF/print output and not the figures as they appear online.
+
+{{< q-class "box tip" >}}
+- This level of CSS refinement should be done as late in the publishing process as possible, as any content changes might shift the way the pages flow and undo the CSS work you did.
+{{< /q-class >}}
+
+#### Change the Figure’s Width
```

Comment:
Rather than "Change the Figure's Width" I wonder if each section should be named after the problem it is resolving?? 

"Reduce White Space by Changing Figure Width"

or something to that effect? 

If I was skimming this section for "copy-paste" solutions, I would start my looking for the problems I want to resolve.

Reply:
Yes, that's better. Updated this and the other headings accordingly